Assignment/Program.cs

```diff
@@ -1,7 +1,18 @@
-﻿namespace Assignment {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
+﻿while (true) {
+   Console.Write ("Enter number : ");
+   if (int.TryParse (Console.ReadLine (), out int num) && num > 0) {
+      int sum = 0;
+      do {
+         while (num > 0) {
+            sum = sum + num % 10;
+            num /= 10;
+         }
+         num = sum;
+         sum = 0;
       }
-   }
+      while (num >= 10);
+      Console.WriteLine ($"Digit root is {num}");
+      break;

```

Comment:
Why we need a infinite loop and break statement.

Reply:
I have used this infinite loop to get valid input from the user.
Also, once the output is achieved then I'm exiting the program, so I have used a break statement to exit the program.

Now I have removed this infinite loop from the program.